qcodes/instrument_drivers/mightex/dllwrapper.py

```diff
@@ -0,0 +1,54 @@
+import ctypes as ct
+import numpy.ctypeslib as npct
+
+''' This module communicates with the SSE SDK dll provided by Mightex. '''
```

Comment:
The module level docstring should be at the very top of the module. Thank you for including one.